src/satosa/backends/apple.py

```diff
@@ -242,78 +110,16 @@ def response_endpoint(self, context, *args):
             raise SATOSAAuthenticationError(context.state, "No user info available.")
 
         all_user_claims = dict(list(userinfo.items()) + list(id_token_claims.items()))
+
+        # convert "string or Boolean" claims to actual booleans
+        for bool_claim_name in ["email_verified", "is_private_email"]:
+            if type(userinfo.get(bool_claim_name)) == str:
+                userinfo[bool_claim_name] = userinfo[bool_claim_name] == "true"
```

Comment:
shouldn't `False` values be set too?

```suggestion
            userinfo[bool_claim_name] = (
                True
                if userinfo[bool_claim_name] == "true"
                else False
            )
```

Reply:
Those are equivalent. `userinfo[bool_claim_name] == "true"` evaluates to `True` or `False`. But your version might be easier to read.